.github/workflows/nightly.yaml

```diff
@@ -41,3 +45,13 @@ jobs:
 
         mv release-debug.yaml nightly-${{ steps.date.outputs.date }}-debug.yaml
         gh release upload nightly nightly-${{ steps.date.outputs.date }}-debug.yaml
+
+        mv sample-strategies.yaml nightly-${{ steps.date.outputs.date }}-sample-strategies.yaml
+        gh release upload nightly nightly-${{ steps.date.outputs.date }}-sample-strategies.yaml
+    - name: Update latest tag of supporting images
+      working-directory: ./cmd
+      run: |
+        for command in *
+        do
+          crane copy "${IMAGE_HOST}/${IMAGE_NAMESPACE}/${command}:nightly-${{ steps.date.outputs.date }}" "${IMAGE_HOST}/${IMAGE_NAMESPACE}/${command}:latest"
```

Comment:
ok totally new to `crane` :-) 

I believe I confirmed that `crane copy` includes the push to the destination:  https://github.com/google/go-containerregistry/blob/2459de30539a35c9514d29b582dea19dbb3330ea/pkg/crane/copy.go#L79

Reply:
Yeah, a nice tool from the go-containerregistry family.